### DISCUSSION TITLE
Documentation Contributions

### DISCUSSION THREAD

**Discussion by aarblaster** [stagehacks/Cue-View#165] [Ideas]
Hi @sparks-alec I'd love to contribute to Cue-View a bit but know 0 JavaScript... I'd be happy to contribute to documentation if you think that's helpful? No dramas if not!

**Comment by sparks-alec** [comment]
I'm gonna be honest and say I did not realize the Discussions area had anything in it because there was no number next to it. Apologies.🤦🏻‍♂️

I am sorry I missed this kind offer from a full 11 months ago! I have turned on public editing of this project's Wiki- you're very encouraged to contribute there if you're still interested 😅

**Reply by aarblaster** [reply]
No worries at all @sparks-alec. Great, thanks.
